dashboard/app/models/user.rb

```diff
@@ -2052,6 +2057,14 @@ def show_census_teacher_banner?
     teacher? && users_school && (next_census_display.nil? || Date.today >= next_census_display.to_date)
   end
 
+  # Returns the name of the donor for the donor teacher banner, or nil if none.
+  def donor_teacher_banner_name
+    school_id = school_info&.school&.id
+    donor_name = DonorSchool.find_by(nces_id: school_id)&.name if school_id
+
+    donor_name
```

Comment:
Right now it seems like we're set up to show this banner to any teacher tied to a school tied to a donor - not specifically to Amazon.  I thought we were expecting to use this table for some other donor-targeted features too, but maybe I misunderstood.

Reply:
There are currently only two columns in the source table: school ID & donor name.  At least with that scheme, the presence of a donor name here is enough to show the banner.  In the future, the banner (in JS) can differentiate its content based on different donor names, but at the moment there is only one donor and the banner is hardcoded for that.